torch_sim/models/orb.py

```diff
@@ -416,14 +410,16 @@ def forward(self, state: SimState | StateDict) -> dict[str, torch.Tensor]:
             _property = "energy" if prop == "free_energy" else prop
             results[prop] = predictions[_property].squeeze()
 
-        if self.conservative:
-            results["direct_forces"] = results["forces"]
-            results["direct_stress"] = results["stress"]
+        if self._conservative:
+            if self.model.forces_name in results:
+                results["direct_forces"] = results[self.model.forces_name]
```

Comment:
If these lines aren't covered by the tests it must mean that these are not defined for conservative models? As such can lines 413-417 be deleted?

Reply:
You are right that the v3 conservative models don't seem to have direct predictions heads as well. I could remove it, but I wanted to make sure that the logic closely follow that of [OrbCalculator](https://github.com/orbital-materials/orb-models/blob/main/orb_models/forcefield/calculator.py#L116-L119).

Reply:
Removed it to make codecov happy!